widget/embedded/src/hooks/useIframe/useIframe.ts

```diff
@@ -0,0 +1,55 @@
+import type { Messages } from '@lingui/core';
```

Comment:
You've brought in the incorrect type declaration for Messages.

Reply:
NIce catch. fixed.